### PR TITLE
Take over Slicer development

### PR DESCRIPTION
**Pull request by eyal0**

### CODE REVIEW

File: _plugins/slicer.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ layout: plugin
 id: slicer
 title: Full-featured Slicer
 description: A full-blown GUI-based slicer. Rotate and scale model; slice multiple STL files at a time; set layer height and other slicing settings.
-author: Kenneth Jiang
+author: eyal0
```
Comment (jneilliii) [inline _plugins/slicer.md:7]: rather than author, shouldn't we make this 
```
authors:
- eyal0
- Kenneth Jiang

**Review by jneilliii — COMMENTED** [review]
@eyal0 I have reached out to @kennethjiang on Discord to see if he would be willing to transfer this repo to your ownership rather than changing over to your fork. This is a little cleaner when it comes to updates and such as we don't have to use the overlays. If I haven't heard back in a couple of days we'll go the overlay route.

Aside from that, I think you should give credit to the original author the way that we have with other plugins, per my code comment.

**Comment by kennethjiang** [comment]
@eyal0 I'm glad that you offered to maintain and further developer the slicer plugin. I have been too busy on TSD to spare any time on the attention this plugin deserves.

I'll be fine with listing @eyal0 as the sole author, or both of us.

File: _plugins/slicer.md (hunk 1 above)
Comment (eyal0) [reply _plugins/slicer.md:7]: Fine by me.  I didn't know that a list was possible there.

**Comment by kennethjiang** [comment]
@eyal0 I have also added you to my original repo. You think it's a good idea to still have the plugin pointing to my original repo, since the stars accumulated in that repo would give the plugin some extra boost in user's confidence. ;)

**Comment by eyal0** [comment]
> @eyal0 I have also added you to my original repo as a collaborator. You think it's a good idea to still have the plugin pointing to my original repo, since the stars accumulated in that repo would give the plugin some extra boost in user's confidence. ;)

That seems good to me, too.  @jneilliii What do you think?  I'll continue the work there and we'll drop this PR?

**Comment by eyal0** [comment]
Also it means that I won't have to copy over all the bug reports.  That's an even better reason not to switch.

**Comment by eyal0** [comment]
I modified this PR.  Please take a look again.

**Comment by kennethjiang** [comment]
lgtm. The only thing I'd say is to put you as the 1st author as you will be actively working on it. :) @eyal0

**Comment by jneilliii** [comment]
Yeah, the order is up to you guys. I could see it both ways, easier to add the new maintainer to the end of the list later down the road, or have current first...your choice. I think for the other plugins that have been adopted it has been new maintainer first if you want to just keep consistency.

**Review by jneilliii — APPROVED** [review]

**Pull request merged**